src/App.js

```diff
@@ -1,26 +1,30 @@
-import React from "react"
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
-import Error from "./Error/Error"
-import Menu from "./Menu/Menu"
-import Home from "./Home/Home"
-import Departments from "./Departments/Departments"
-import Department from "./Department/Department"
+import React, { Suspense } from 'react'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import PageError from './PageError/PageError'
+import PrivateRoute from './PrivateRoute/PrivateRoute'
 
+const Login = React.lazy(() => import ('./Login/Login'))
+const Departments = React.lazy(() => import ('./Departments/Departments'))
+const Department = React.lazy(() => import ('./Department/Department'))
+// const Employees = React.lazy(() => import ('./Employees/Employees'))
+// const Employee = React.lazy(() => import ('./Employee/Employee'))
 
 function App() {
   return (
     <Router>
-      <div className="App">
-        <Menu/>
-        <Switch>
-          <Route exact path="/" component={Home}/>
-          <Route path="/departments/:id" component={Department}/>
-          <Route path="/departments" component={Departments}/>
-          <Route path="/*" component={Error}/>
-        </Switch>
+      <div className='App'>
+          <PrivateRoute exact path='/' />
+          <Suspense fallback={<div>Loading</div>}>
+            <Switch>
+              <Route path='/login' component={Login} />
```

Comment:
login should not be used under suspense

Reply:
Can you explain me the different? Because if my Login component is higher, not in Suspense, ErrorPage is becoming active. Like on printscreen
![1](https://user-images.githubusercontent.com/12400717/155172093-6b996321-3826-4207-95de-9bcf08354d70.png)

and here Login under Suspense
![2](https://user-images.githubusercontent.com/12400717/155172206-ed25f425-71d9-4069-b547-edabb6f1f6fd.png)

Reply:
because you put route outside of switch